libp2p/src/tutorials/hole_punching.rs

```diff
@@ -122,16 +123,17 @@
 //!
 //! ``` bash
 //! ## Inside the rust-libp2p repository.
-//! cargo build --example client -p libp2p-dcutr
+//! cd ./examples/relay-server
+//! cargo build
```

Comment:
Seems to be a copy-paste error here. I'd also recommend that we specify the binary directly:

```suggestion
//! cargo build --bin dcutr
```